### PR TITLE
LPS-45616 SF

### PR DESCRIPTION
**Pull request by brunobasto**

### CODE REVIEW

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js [hunk 1]
```diff
@@ -485,6 +493,111 @@ AUI.add(
...
+
+						return value;
+					},
+
+					_setFrequency: function(value) {
+						var instance = this;
+
+						var frequencySelect = instance.get('frequencySelect');
+
+						frequencySelect.set('value', value);
```
Comment (brunobasto) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js:517]: @fernandosouza We should not be using .simulate() in production code. It doesn't work very well. See: http://yuilibrary.com/yui/docs/event/simulate.html. Read the section "Don't use simulation in user facing code".

Can you write it in another way and resend? Thanks,

Comment (fernandosouza) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js:517]: Ok Bruno.

**Comment by fernandosouza** [comment]
Just started reviewing :) <br><br>:octocat: _Sent from [GH](http://nodegh.io)._

File: modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js (hunk 1 above)
Comment (fernandosouza) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js:517]: cc @brandizzi

Comment (fernandosouza) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js:517]: @brandizzi. I am working on a refactoring of the _onInputChange method. Maybe I will need your help to test it. OK?

Comment (brandizzi) [inline modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/recurrence.js:517]: Sure! Count on me. Just ping me whenever you need.

**Pull request closed without merging**